src/main/java/org/quantumbadger/redreader/fragments/AccountListDialog.java

```diff
@@ -49,9 +49,6 @@
 
 	private AppCompatActivity mActivity;
 
-	// Workaround for HoloEverywhere bug?
```

Comment:
I think it would be better to leave this in. The app doesn't use HoloEverywhere any more, but I never verified that that was the cause of the issue.